static/js/tools/map/fetcher/breadcrumb_denom_stat.ts

```diff
@@ -50,11 +50,12 @@ export function useFetchBreadcrumbDenomStat(
       type: ChartDataType.BREADCRUMB_DENOM_STAT,
       context: {
         placeInfo: {
-          enclosingPlace: {
+          selectedPlace: {
             // entities are the parent places + the enclosing place.
             // so the last element is the enclosing place.
             dcid: placeDcids[placeDcids.length - 1],
             name: "",
+            types: null,
```

Comment:
curious why we add types for selectedPlace here but not in the default_stat fetcher or any other fetcher

Reply:
selectedPlace uses the type "NamedTypedPlace" which requires types. This maybe due to some other map logic?